python/hail/methods/impex.py

```diff
@@ -354,18 +380,17 @@ def import_locus_intervals(path, reference_genome='default', skip_invalid_interv
 
     Note
     ----
+    Files with one field **AND** `reference_genome` is defined are parsed with
```

Comment:
This sentence needs a little cleanup. Maybe "If `reference_genome` is defined **AND** the file has one field, intervals are parsed with :func:`.parse_locus_interval`."